### PR TITLE
Remove duplicated words in files

### PR DESCRIPTION
**Pull request by mooncak**
Remove duplicated words in files:

1. CLA.md
2. contributors/design-proposals/node/accelerator-monitoring.md

### CODE REVIEW

File: CLA.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ You must also set your [git e-mail](https://help.github.com/articles/setting-you
 to match this e-mail address as well.
 
 If you already submitted a PR you can correct your user.name and user.email
-and then use use `git commit --amend --reset-author` and then `git push --force` to
```
Comment (duglin) [inline CLA.md]: I can't seem to find it right now but I think I saw this fix in another PR - guess we'll see which gets in first :-)

Comment (cblecker) [reply CLA.md]: It was already in https://github.com/kubernetes/community/commit/d1dbb3cb217557d93e128a0bfcac58a6e6554851.

**Comment by mooncak** [comment]
close this PR as it's already fixed.

**Pull request closed without merging**